docs/consume-packages/Central-Package-Management.md

```diff
@@ -9,79 +9,110 @@ ms.topic: conceptual
-use the latest compatible tooling versions.
+Older tooling will ignore Central Package Management configurations and features.
+To use this feature to the fullest extent, ensure all your build environments use the latest compatible tooling versions.
 
-Central package management applies to all `<PackageReference>`-based MSBuild projects (including
-[legacy CSPROJ](https://github.com/dotnet/project-system/blob/main/docs/feature-comparison.md)) as long as compatible tooling is used.
+Central Package Management applies to all `<PackageReference>`-based MSBuild projects (including [legacy CSPROJ](https://github.com/dotnet/project-system/blob/main/docs/feature-comparison.md)) as long as compatible tooling is used.
 
 ## Enabling Central Package Management
 
-To get started with central package management, you must create a `Directory.Packages.props` file at the root of your repository and set the MSBuild property
-`ManagePackageVersionsCentrally` to `true`.
+To get started with Central Package Management, create a `Directory.Packages.props` file at the root of your repository and set the MSBuild property `ManagePackageVersionsCentrally` to `true`.
+
+You can create it manually, or use the .NET CLI:
 
-You can create it manually or you can use the dotnet CLI:
-``` shell
+```shell
 dotnet new packagesprops
 ```
 
-Inside, you then define each of the respective package versions required of your projects using `<PackageVersion />` elements that define the package ID and
-version.
+Inside `Directory.Packages.props`, define `<PackageVersion />` elements to specify the package IDs and versions used by your projects.
 
 ```xml
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageVersion Include="PackageA" Version="1.0.0" />
+    <PackageVersion Include="PackageB" Version="2.0.0" />
   </ItemGroup>
 </Project>
 ```
 
-For each project, you then define a `<PackageReference />` but omit the `Version` attribute since the version will be obtained from a corresponding
-`<PackageVersion />` item.
+In each project file, define `<PackageReference />` elements without the `Version` attribute.
+The version will be resolved from the corresponding `<PackageVersion />` entry in `Directory.Packages.props`.
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="PackageA" />
   </ItemGroup>
 </Project>
 ```
 
-Now you're using central package management and managing your versions in a central location!
+Now you're using Central Package Management and managing your versions in a central location!
 
-## Central Package Management rules
+### Using Different Versions for Different Target Frameworks
```

Comment:
I wonder if the order of this section and the rules one should be switched.

Reply:
I'll move it down after the Getting Started section with the other stuff that is related to this kind of content.